bastionzero/zli
# Windows agent autodiscovery

## Description of the change

  - Add `zli generate powershell` command to autodiscover Windows targets
  - Remove `--agentVersion` from `generate bash` since we can't support it with bzero targets
  - Add `--beta` to both commands since we can support that

## Testing

  - Check out this branch and the corresponding backend branch
  - `zli generate powrshell` with any options you please
  - Try spinning up a new Windows EC2 and using the generated script as user data
  - After a couple minutes you'll see it register with your dev stack

**backend branch:** feat/windows-autodiscovery
**bzero branch:** 
**charts branch:** 

#### Ready to run system tests?


## Relevant release note information

Release Notes: Add `zli generate powershell` command to autodiscover Windows targets

## Related JIRA tickets

Relates to JIRA: CWC-2674

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?


If yes, please explain:

Does this PR add a new/modifies the name/arguments of a `zli` command?


If yes, please provide the docs/zli-reference-manual branch reflecting the changes:
